include/alpaka/idx/bt/IdxBtCudaBuiltIn.hpp

```diff
@@ -104,11 +103,11 @@ namespace alpaka
                 template<
                     typename TWorkDiv>
                 ALPAKA_FN_ACC_CUDA_ONLY static auto getIdx(
-                    idx::bt::IdxBtCudaBuiltIn<TDim, TIdx> const & /*idx*/,
+                    idx::bt::IdxBtCudaBuiltIn<TDim, TIdx> const & idx,
                     TWorkDiv const &)
                 -> vec::Vec<TDim, TIdx>
                 {
-                    //alpaka::ignore_unused(idx);
+                    alpaka::ignore_unused(idx);
```

Comment:
Why is this solved with `ignore_unused` and not the standard approach with an unnamed parameter (which is now `idx`)?

Reply:
Because the name is information that would else be lost especially in a header inly library. Furthermore this requires ifdefs in some cases.

Reply:
Furthermore, you need a name to add a doxygen description.

Reply:
@BenjaminW3 I see. However, to be fair it is not because the library is header-only, but only because the function is defined right at declaration. Otherwise, the declaration will have a parameter and implementation won't, which is logical for both. (Also I understand the value of ```ignore_unused``` when smth is conditionally used, which is not the case here).

Reply:
yes, you are both right.